test/integration/appsource/assets/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  labels:
+    application: cymbal-bank
+    environment: development
+    team: frontend
+    tier: web
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: frontend-ip
```

Comment:
Thanks @amandakarina! I diver deeper later today or tomorrow, but does this conflict with the existing `Gateway` in the production environment? https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/6-appsource/cymbal-bank/frontend/k8s/overlays/production/gateway.yaml#L36

Upon further thought I know we want to use Multi-Cluster `Gateway` for production (https://cloud.google.com/architecture/enterprise-application-blueprint/cymbal-bank), so it might be simpler to use `Gateway` for all environments similar to how you have this `Ingress`?

Reply:
Currently, only dev env is being tested, and the ingress is being copied to dev only when deploying in 6-appsouce

Reply:
Ahh, good to know - I'd forgotten that only dev is currently being tested.